po/de.po

```diff
@@ -5125,7 +5125,7 @@ msgid ""
 "Matches people that have no family relationships to any other person in the "
 "database"
 msgstr ""
-"Liefert Personen, die keine familiären Beziehungen zu irgend jemandem in der "
+"Liefert Personen, die keine familiären Beziehungen zu irgendjemandem in der "
```

Comment:
```suggestion
"Liefert Personen, die keine familiären Beziehungen zu jemandem in der "
```
sounds even better and also gets the right point, I think.

Reply:
You are quite right, but in this PR I focused on other things.

There are more corrections to come in the future. :-)